server/src/stateManagement.mjs

```diff
@@ -365,7 +366,10 @@ function validateNewState_MethodOverrides(newStateMethods) {
   for ( const [methodName, methodOverrideObject] of Object.entries(newStateMethods) ) {
     errors = errors.concat(validateMethodOverride(methodName, methodOverrideObject));
   }
-
+// returns an empty array in "novalidate mode"
+  if( !validateMethodOverrides ){
```

Comment:
You need to check both the command-line setting here AND the dotconfig object flag. What you have right now ignores anything in the dotconfig file, I think.

Reply:
Also... you should do this check and possible return at the top of the function; no need to do all the validation only to throw away.

Reply:
This check I have taken care in commandLine.mjs:
if( !config.dotConfig.validateFlag && parsed.novalidate ){
  validateMethodOverrides = false;
}
Please confirm.

Reply:
Done, Please confirm.